x-pack/test/security_solution_cypress/cypress/tasks/entity_analytics.ts

```diff
@@ -113,3 +118,17 @@ export const upgradeRiskEngine = () => {
   updateRiskEngineConfirm();
   cy.get(RISK_SCORE_STATUS).should('have.text', 'On');
 };
+
+export const openEntityStoreEnablementModal = () => {
+  cy.get(ENTITY_STORE_ENABLEMENT_BUTTON).click();
+  cy.get(ENTITY_STORE_ENABLEMENT_MODAL).contains('Entity Analytics Enablement');
```

Comment:
For future reference, is it ok to have the assertions outside of the task :)  

The idea of wrapping the interactions is just to improve the readability and the intent of the test.